.github/workflows/ci.yml

```diff
@@ -265,51 +209,15 @@ jobs:
     strategy:
       matrix:
         python-version: [ "3.8","3.9","3.10","3.11" ]
-    runs-on: ubuntu-latest-m
+    runs-on: ubuntu-latest
```

Comment:
do we know why it was switched to the larger size and why it's okay to switch it back now?

Reply:
It shouldn't need the large size for setup since it's not using multiple processes to pull in the dependencies. This may have been done in error (by me) when the instance size was being increased for ingest related CI steps.